src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -392,19 +382,21 @@ public void testCLTVPaymentChannelOutput() {
     public void getToAddress() {
         // pay to pubkey
         BtcECKey toKey = new BtcECKey();
-        Address toAddress = toKey.toAddress(PARAMS);
-        assertEquals(toAddress, ScriptBuilder.createOutputScript(toKey).getToAddress(PARAMS, true));
+        Address toAddress = toKey.toAddress(mainnetParams);
+        assertEquals(toAddress, ScriptBuilder.createOutputScript(toKey).getToAddress(
```

Comment:
Could we put the value from the function in a separate variable to get to know the value before the assert when we are debugging the code?

Reply:
Updated.